src/data/indexHostSingleton.ts

```diff
@@ -34,19 +29,17 @@ export const IndexHostSingleton = (function () {
     }
   };
 
-  const key = (config, indexName) => `${config.apiKey}-${indexName}`;
+  const _key = (config: PineconeConfiguration, indexName: string) =>
```

Comment:
stylistic change, underscore just to make it more clear it's internal to the singleton